src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/project.json

```diff
@@ -19,6 +19,8 @@
     "System.Threading": "4.4.0-beta-24619-02",
     "System.Threading.Thread": "4.4.0-beta-24619-02",
     "System.Threading.Tasks": "4.4.0-beta-24619-02",
+    "System.Xml.ReaderWriter": "4.4.0-beta-24619-02",
+    "System.Xml.XPath": "4.4.0-beta-24619-02",
```

Comment:
If these tests depend on System.Runtime.Loader then they should be converted to be netcoreapp1.1 specific and then you should be able to remove this project.json.